### PR TITLE
Add account validator import command

### PR DESCRIPTION
**Pull request by dapplion**
Allows to import keystores generated with the EF Eth2.0 Launchpad. Implements https://github.com/sigp/lighthouse/blob/master/account_manager/src/validator/import.rs

Prompts for each Keystore password with inquirer `password` (hidden) question type.

Sample output of the command

```
lion@lion-gram:~/Code/eth2.0/lodestar/packages/lodestar-cli$ ts-node src/ account validator import --directory ~/Downloads/

/home/lion/Downloads/keystore-m_12381_3600_0_0_0-1595959302.json
/home/lion/Downloads/validator1/keystore-m_12381_3600_0_0_0-1595959302.json

Found 2 keystores in 	/home/lion/Downloads/
Importing to 		.lodestar/keystores


Keystore found at /home/lion/Downloads/keystore-m_12381_3600_0_0_0-1595959302.json
- Public key: 0xb709108cf222c87d64526c393d872961f647f438b483365c14e5c0a26d08862cf06d10e630a71816c1920cb8ac699260
- UUID: deba008c-3b56-4f49-981b-0e62fd6c0171

If you enter the password it will be stored as plain-text in .lodestar/secrets so that it is not required each time the validator client starts.

? Enter the keystore password, or press enter to omit it [hidden]
Password is correct
Successfully imported validator b709108cf222c87d64526c393d872961f647f438b483365c14e5c0a26d08862cf06d10e630a71816c1920cb8ac699260
Skipping existing validator b709108cf222c87d64526c393d872961f647f438b483365c14e5c0a26d08862cf06d10e630a71816c1920cb8ac699260

Successfully imported 1 validators (1 skipped)

DO NOT USE THE ORIGINAL KEYSTORES TO VALIDATE WITH
ANOTHER CLIENT, OR YOU WILL GET SLASHED.
```

### CODE REVIEW

File: packages/lodestar-cli/src/cmds/account/cmds/validator/import.ts [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+  writeFile600Perm,
+  recursivelyFindVotingKeystores
+} from "../../../../util";
+import {VOTING_KEYSTORE_FILE} from "../../../../validatorDir/paths";
+import {IAccountValidatorOptions} from "./options";
+import {getAccountPaths} from "../../paths";
+
+export const command = "import";
+
+export const description = "Imports one or more EIP-2335 passwords into a Lodestar validator client directory, \
```
Comment (mpetrunic) [inline packages/lodestar-cli/src/cmds/account/cmds/validator/import.ts]:
```suggestion
export const description = "Imports one or more EIP-2335 keystores into a Lodestar validator client directory, \
```
?

File: packages/lodestar-cli/src/cmds/account/cmds/validator/import.ts [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+    console.log(`
+Keystore found at ${keystorePath}
+- Public key: 0x${pubkey}
+- UUID: ${uuid}
+
+If you enter the password it will be stored as plain-text in ${secretsDir} so that it is not \
+required each time the validator client starts.
+`);
+    
+    const answers = await inquirer.prompt<{password: string}>([{
```
Comment (mpetrunic) [inline packages/lodestar-cli/src/cmds/account/cmds/validator/import.ts]: Should we check for .pass file before asking for password?

File: packages/lodestar-cli/src/cmds/account/cmds/validator/import.ts [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+          if (input) keystore.decrypt(stripOffNewlines(input));
+          return true;
+        } catch (e) {
+          return `Invalid password: ${e.message}`;
+        }
+      }
+    }]);
+
+    console.log("Password is correct");
+    await new Promise(r => setTimeout(r, 1000)); // For UX
```
Comment (mpetrunic) [inline packages/lodestar-cli/src/cmds/account/cmds/validator/import.ts]: I think you have "sleep" function in utils to make it more verbose

Comment (dapplion) [reply packages/lodestar-cli/src/cmds/account/cmds/validator/import.ts]: There's one in `packages/lodestar-validator`, should we import from there?

File: packages/lodestar-cli/src/cmds/account/cmds/validator/import.ts (hunk 2 above)
Comment (dapplion) [reply packages/lodestar-cli/src/cmds/account/cmds/validator/import.ts]: Sure

File: packages/lodestar-cli/src/cmds/account/cmds/validator/import.ts (hunk 3 above)
Comment (mpetrunic) [reply packages/lodestar-cli/src/cmds/account/cmds/validator/import.ts]: not really, oh well

**Review by wemeetagain — APPROVED** [review]
:tada:

**Pull request merged**